src/main/java/org/spongepowered/api/block/data/Sign.java

```diff
@@ -37,7 +37,7 @@
      *
      * @return The lines of text
      */
-    Message[] getLines();
+    Text[] getLines();
```

Comment:
I agree with simon here. In addition to that we do not use arrays as return types very often or at all in the API. So for consistency reason this should be a list.